src/Aevatar.MongoDB/Notification/NotificationDbContext.cs

```diff
@@ -0,0 +1,10 @@
+using Aevatar.ApiKey;
+using MongoDB.Driver;
+using Volo.Abp.MongoDB;
+
+namespace Aevatar.Notification;
+
+public class NotificationDbContext : AbpMongoDbContext
```

Comment:
Maybe there's no need to create a NotificationDbContext, AevatarMongoDbContext is enough?